### PR TITLE
:bug: fixes the 10MB limit in store

### PR DESCRIPTION
**Pull request by byronz**
Post-merge checklist:

The master branch is auto-deployed to `dash.plot.ly`.
Once you have merged your PR, wait 5-10 minutes and check dash.plot.ly 
to verify that your changes have been made.

- [x] I understand

### CODE REVIEW

File: tutorial/core_component_examples.py [hunk 1]
```diff
@@ -1258,8 +1258,13 @@
     - `modified_timestamp` is read only.
 
     ### local/session specifics
+
     - The store will not work properly if it's not included in the initial layout.
-    - The total data of all stores should not exceed 10MB.
+    - The maximum browser [storage space](https://arty.name/localstorage.html) is determined by the following factors:
+        - your own hard drive size
+        - mobile or laptop
+        - the browser, under which a sophiticated algorithm is implmented within *Quota Management*
```
Comment (alexcjohnson) [inline tutorial/core_component_examples.py]: Can we say something about what typical values are? What's safe for all users? What's safe if your users are all on modern laptops?

Comment (byronz) [reply tutorial/core_component_examples.py]: sth like these tables? https://www.html5rocks.com/en/tutorials/offline/quota-research/

Comment (alexcjohnson) [reply tutorial/core_component_examples.py]: FWIW at that test site above, on this Mac, I get 2.6M characters in Safari, 5.2M in Chrome. My phone (Safari under the hood) also gives 2.6M

Comment (byronz) [reply tutorial/core_component_examples.py]: yes, I also have 5M in chrome, but source everywhere was talking about 10MB

**Pull request merged**